### ISSUE TITLE
Build failing on pyslim

### ISSUE THREAD

**Issue by jeromekelleher** [tskit-dev/tskit-site#171]
https://github.com/tskit-dev/tskit-site/actions/runs/4252203086/jobs/7395493382

**Comment by benjeffery** [comment]
Same issue as the rest of the repos. Will fix.

**Comment by petrelharp** [comment]
Unhelpful notes:

Currently failing:
- do docs build locally without `jsonschema[format-nongpl]==4.17.3` and `tskit-book-theme`? *yes*
- do docs build locally without `jsonschema[format-nongpl]==4.17.3` but with `tskit-book-theme`? *no*

So, looks like
```
tskit-book-theme 0.3.2 requires pydata-sphinx-theme~=0.8.0, but you have pydata-sphinx-theme 0.12.0 which is incompatible.
sphinx-book-theme 1.0.1 requires pydata-sphinx-theme>=0.13.3, but you have pydata-sphinx-theme 0.12.0 which is incompatible.
...
jupyter-book 0.15.1 requires sphinx-book-theme~=1.0.0, but you have sphinx-book-theme 0.3.3 which is incompatible.
```
the problem is fundamentally an incompatibility between tskit-book-theme and certain versions of jupyter-book? Does anyone know what it'd take to upgrade tskit-book-theme?

**Comment by petrelharp** [comment]
So, I tried [copying over](https://github.com/tskit-dev/pyslim/actions/runs/5350864935/jobs/9704012106?pr=313) the requirements.txt file from tskit, and got this error:
```
 ERROR: Cannot install -r requirements/CI-docs/requirements.txt (line 46) and jsonschema==4.17.3 because these package versions have conflicting dependencies.

The conflict is caused by:
    The user requested jsonschema==4.17.3
    jupyter-book 0.13.1 depends on jsonschema<4
```
... oddly, since it seemed to work for tskit?

**Comment by benjeffery** [comment]
Fixed in #176
